.github/workflows/build_documentation.yml

```diff
@@ -1,6 +1,7 @@
 name: Build documentation
 
 on:
+  workflow_dispatch:
```

Comment:
My only concern is if `${{ github.sha }}` (see below) will be the correct (previous) release tag commit.

Reply:
According to Github doc here: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch

> Last commit on the GITHUB_REF branch or tag

So I think it should work fine.